rcl_yaml_param_parser/src/parser.c

```diff
@@ -1105,7 +1106,22 @@ static void * get_value(
   {
     errno = 0;
     endptr = NULL;
-    dval = strtod(value, &endptr);
+    const char * vernier_ptr = NULL;
```

Comment:
Just a question: what's a `vernier_ptr`?  I haven't heard that term before.

Reply:
> I haven't heard that term before.

Me neither :). The pointer used to iterate the `value` a bit like a cursor or something, so i named it like this. Now, i prefer to name it `iter_ptr`, which i think it's more appropriate.